Frontend/src/overrides.css

```diff
@@ -1,5 +1,16 @@
-/* We don't want to have the padding that is caused by having two containers */
-.registration-container {
+/* We don't want to have the padding that is caused by having two containers in admin mode */
+.container-fluid > #registration-app > .registration-container {
```

Comment:
Where is the class `.container-fluid` coming from?
I always thought that `.container` and `.fluid` are two separate classes

Reply:
It's from the monolith. It's because in admin mode there is an extra container-fluid from the monolith's application css

Reply:
Ah, yikes!
Well I guess as long as it works... Monolith UI is deprecated so I don't want to touch that code over there too much :(